frontend/src/components/navigation/__snapshots__/Header.test.tsx.snap

```diff
@@ -24,7 +24,7 @@ exports[`header renders correctly 1`] = `
           >
             <a
               className="navbar-brand"
-              href="https://gov.bc.ca"
+              href="http://localhostnull"
```

Comment:
As you can see in this snapshot your code above should not be concatenating `null` to the string. It will get "null" appended!

Reply:
fixed